commodities/models/orm.py

```
@@ -362,6 +362,9 @@ def get_parent_node(
             .first()
         )
 
+        if not parent:
+            return
+
```

Should this ever be true if we've gone past the "root node" check?? It might hide errors?

---

parasite from a test i was doing! removing